recipe/meta.yaml

```
@@ -1,19 +1,15 @@
 {% set name = "arcticdb" %}
-{% set version = "4.0.0rc0" %}
+{% set version = "4.0.0rc1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/man-group/ArcticDB/archive/refs/tags/v{{ version }}.tar.gz
-  sha256: 0b48cd344d630e5f2c23b13898f6b022f3285875d4a269badcc9cd0c4df34ac6
+  sha256: 3e2bbd667f1cf198956219590752476bb59aecfd11e580a34f19340debe33401
 
 build:
```

We still should skip Windows for the builds, but update the comment which is currently out-of-date (my bad). Currently, the builds on Windows is blocked by an issue with linking symbols for libprotocol, see https://github.com/man-group/ArcticDB/pull/449#issuecomment-1622137542.

```suggestion
build:
  # We skip the build on Windows because of linkage problems with libprotobuf
  # See: https://github.com/man-group/ArcticDB/pull/449
  skip: true  # [win]
```